tools/data-collect/aus-viewer.py

```diff
@@ -0,0 +1,316 @@
+                                    "Temporal Filter",
+                                    "Threshold Filter"
+                                    ]
+
+        },
+        'sensors_used_per_device': { 'Color': {
+                                        "Intel RealSense D435I":[['11:34:42','11:37:43'], ['11:39:42','11:41:24']]
+                                            ,"Intel RealSense D457":[['11:34:42','11:37:43'], ['11:39:42','11:41:24']],
+                                            "Intel RealSense D415":[['11:34:42','11:38:24'], ['11:39:42','11:41:24']]
+
+        },
+            'Accel': {
+                "Intel RealSense D435I":[['11:34:42','11:38:24'], ['11:39:42','11:41:24'] ]
+                , "Intel RealSense D457":[['11:34:42','11:37:43'], ['11:39:42','11:41:24']],
+            },
+            'Gyro' : {
+                "Intel RealSense D435I":[['11:34:42','11:38:43']],
+                    "Intel RealSense D415":[['11:34:42','11:38:24'], ['11:39:42','11:41:24']]
+
+            },
+            'Depth': {
+                "Intel RealSense D435I":[['11:34:42','11:34:51']],
+                "Intel RealSense D415":[['11:34:42','11:38:24'], ['11:39:42','11:41:24']]
+            }
+        }
+
+    }
+
+ppf_list = ["Decimation Filter", "Depth to Disparity", "Disparity to Depth", "HDR Merge Filter", "Hole Filling Filter", "Filter By Sequence id", "Spatial Filter", "Temporal Filter", "Threshold Filter"]
+
+
+class AUS(QMainWindow):
+    def __init__(self):
+        super().__init__()
+        self.title = 'RealSense AUS Viewer'
+        logo_path = os.path.join(os.getcwd(), 'icon.ico')
+        logo = QIcon(logo_path)
+        self.setWindowIcon(logo)
+        self.left = 10
+        self.top = 100
+        self.width = 540
+        self.height = 550
+        central_widget = QWidget(self)
+        self.setCentralWidget(central_widget)
+        self.initUI()
+
+    def initUI(self):
+        grid = QGridLayout(self.centralWidget())
+        grid.addWidget(self.static_data_and_devices(), 0, 0)
+        grid.addWidget(self.add_timers_data(), 0, 2)
+        grid.addWidget(self.ppf_per_device(), 0, 1)
+
+        self.setLayout(grid)
+        self.setWindowTitle(self.title)
+        self.setGeometry(self.left, self.top, self.width, self.height)
+        self.show()
+
+    def static_data_and_devices(self):
+        groupBox = QGroupBox("Static data")
+        groupBox.setFont(QFont('Times', 8))
+        self.add_static_data_labels()
+
+        self.chart_counters = QChart()
+        self.chart_view_counters = QChartView(self.chart_counters)
+        self.chart_view_counters.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Expanding)
+        self.chart_view_counters.setMinimumSize(500, 500)
+        self.chart_counters.setTheme(QChart.ChartThemeQt)
+        self.chart_counters.setBackgroundBrush(QBrush(QColor("transparent")))
+        self.add_pie_chart_data()
+
+        vbox = QVBoxLayout()
```

Comment:
in future commits, we should separate these styling and gui implementations to multiple files, in addition to files that are more like a "backend" that fills and manipulate the inputs.
For POC it is enough and okay. Just keep in mind.